homestar-runtime/tests/metrics.rs

```diff
@@ -55,9 +51,9 @@ fn test_metrics_serial() -> Result<()> {
         .stdout(Stdio::piped())
         .spawn()
         .unwrap();
+    let _guard = ChildGuard::new(homestar_proc);
```

Comment:
Not observing one, but is there any possible issue with `_guard` shadowing the file guard with the same name a few lines up?

Reply:
Nope.